read-fonts/src/tables/layout.rs

```diff
@@ -101,6 +101,7 @@ impl RangeRecord {
     }
 }
 
+#[allow(clippy::derivable_impls)]
```

Comment:
If we can derive ti ... shouldn't we?

Reply:
the type is declared in generated code, so we would need syntax in codegen for specifying default values, which we don't have.